runtimes/runtimes/operational-telemetry/operational-event-validator.ts

```diff
@@ -0,0 +1,25 @@
+import Ajv from 'ajv'
+
+export class OperationalEventValidator {
+    private readonly ajv
+    private readonly validate
+
+    constructor() {
+        this.ajv = new Ajv()
+
+        this.ajv.addSchema(require('./telemetry-schemas/base-event-schema.json'), 'base-event-schema.json')
```

Comment:
Can be more generic by iterating through all files in `telemetry-schemas` folder.